examples/incompressible-flow/2d-nitsche-taylor-couette/adaptative-nitsche-taylor-couette.prm

```diff
@@ -150,10 +150,11 @@ end
 subsection linear solver
   subsection fluid dynamics
     set verbosity                                 = verbose
-    set method                                    = amg
+    set method                                    = gmres
     set max iters                                 = 500
     set relative residual                         = 1e-3
     set minimum residual                          = 1e-10
+    set preconditioner                            = amg
     set ilu preconditioner fill                   = 3
     set ilu preconditioner absolute tolerance     = 1e-12
     set ilu preconditioner relative tolerance     = 1.00
```

Comment:
That's true. Basically, the parameters defined for the ilu preconditioner are not being used and should be removed. Only the amg parameters specified below are taken into account. So, I will remove them so that it is clean and clear.